policyengine_us/parameters/gov/states/ar/tax/income/credits/inflationary_relief/reduction/amount.yaml

```diff
@@ -0,0 +1,25 @@
+description: Arkansas reduces the inflation relief credit by this amount for each increment of adjusted gross income above the threshold, based on the filing status.
+metadata:
+  period: year
+  unit: currency-USD
+  label: Arkansas inflation relief credit reduction amount
+  reference:
+    # The inflation relief credit is not mentioned in the Arkansas statutes 
+    - title: Arkansas 2022 Individual Income Tax Forms and Instructions 
+      href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=26
+    - title: INFLATIONARY RELIEF INCOME-TAX CREDIT WORKSHEET | page1
+      href: https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_Final_InflationaryReliefIncomeTaxCredit_FI.pdf
```

Comment:
add house bill: 
https://www.arkleg.state.ar.us/Home/FTPDocument?path=%2FAssembly%2F2021%2F2022S3%2FFiscal+Impacts%2FHB1002-DFA1.pdf#page=2

Reply:
in reduction amount and increment add comment: 
# House bill did not specify reduction amount / increment